frontend/components/employee/index.js

```diff
@@ -0,0 +1,108 @@
+    const [rowSelection, setRowSelection] = useState([]);
+
+    const closeAddOpen = () => {
+        setAddOpen(false);
+    };
+
+
+    const handleSearch = (val) => {
+        let searchValue = val.toLowerCase().trim()
+        if (searchValue.length > 0) {
+            let temp = employeeList.filter(
+                (emp) =>
+                    emp.first_name.toLowerCase().includes(searchValue) ||
+                    emp.last_name.toLowerCase().includes(searchValue)
+            );
+            console.log(temp)
+            setDisplayEmployeeList(temp);
+        } else {
+            setDisplayEmployeeList(employeeList);
+        }
+    };
+    const columns = [
+        {field: 'first_name', headerName: 'First name', width: 250, sortable: false},
+        {field: 'last_name', headerName: 'Last name', width: 300, sortable: false},
+        {field: 'title', headerName: 'Title', width: 250, sortable: false},
+        {field: 'email', headerName: 'Email', width: 300, sortable: false},
+
+    ];
+    const handleDeleteEmployee = () => {
+        console.log(rowSelection)
+        deleteEmployee(rowSelection[0]);
+    }
+
+
+    return (
+
+        <Box>
+            <SearchInput handleSearch={handleSearch}/>
+
+            <Grid
+                container
+                direction="row"
+                justifyContent="flex-start"
+                alignItems="center"
+            >
+                <Grid item xs={6}>
+                    <h1>Employee List
+                    </h1>
+                </Grid>
+                <Grid item xs={3}>
+                    <Button className={styled.addButton} variant="outlined" onClick={() => setAddOpen(true)}>
+                        New Employee
+                    </Button>
+                </Grid>
+                <Grid item xs={3}>
+                    <Button className={styled.addButton} disabled={!(rowSelection.length > 0)} variant="outlined"
+                            onClick={handleDeleteEmployee}>
+                        Delete Employee
+                    </Button>
+                </Grid>
+
+
+            </Grid>
+            <Divider/>
+            <DataGrid
+                style={{minHeight: '60%', height: '560px'}}
+                rows={employeeList}
+                columns={columns}
+                pagination
+                pageSize={8}
+                onRowClick={({row}) =>
+                    router.push({
+                        pathname: '/employee/details',
+                        query: {empid: row.id}
+                    }, '/employee')
+                }
+                rowsPerPageOptions={[8]}
+                checkboxSelection
+                hideFooterSelectedRowCount
+                disableColumnMenu
+                selectionModel={rowSelection}
+                onSelectionModelChange={(rows) => setRowSelection(rows)}
+            />
+
+            <NewEmployeeDialog open={addOpen} handleClose={closeAddOpen} addEmployee={addEmployee}
```

Comment:
Since `closeAddOpen` is used only once, I suggest replacing it with arrow functions like line 68.

Reply:
Ok, it is replace with arrow function.